bitmap_test.go

```
@@ -0,0 +1,136 @@
+package main
+
+import (
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestNewBitmap(t *testing.T){
+	tests:= []struct{
+		input int
+		output Bitmap
+
+	}{
+		{input:7 , output:Bitmap{size : 7,bitArray : make([] uint64, 1,1)}},
+		{input:65 , output:Bitmap{size : 65,bitArray : make([] uint64, 2,2)}},
+		{input:-5 , output:Bitmap{size : 0,bitArray : make([] uint64,0,0)}},
+	}
+	for _,tc := range tests{
+		r,_:= NewBitmap(tc.input)
```

Don't ignore the error. assert if it is nil